pkg/config/config.go

```diff
@@ -34,6 +34,9 @@ const (
 	ExtAuthzTimeout          = "KOURIER_EXTAUTHZ_TIMEOUT"
 	ExternalAuthzCluster     = "extAuthz"
 
+	DuplicatedDomainReason  = "DomainConflict"
+	DuplicatedDomainMessage = "Ingress rejected as its domain conflicts with another ingress"
```

Comment:
This doesn't feel like config 🤔 , since these values are unused, shall we just inline the strings for now?